go.mod

```diff
@@ -209,6 +206,7 @@ require (
 	github.com/klauspost/pgzip v1.2.5 // indirect
 	github.com/kr/pretty v0.3.1 // indirect
 	github.com/kr/text v0.2.0 // indirect
+	github.com/kyverno/kyverno v1.9.2 // indirect
```

Comment:
why do we still have kyverno dependency?

Reply:
this is presnt as an indirect dependency due to `discovery-engine` package, which uses `kyverno` pkg

Reply:
We will need to wait for to remove summary to eliminate it ig. Thanks